translation.rst

```diff
@@ -264,6 +264,10 @@ The ``translation:update`` command looks for missing translations in:
 
     If you want to see the missing translation strings without actually updating
     the translation files, remove the ``--force`` option from the command above.
+    
+.. note::
+
+    If you want to see translation output strings sorted alphabetically, add ``--sort=asc`` option. Or you can use ``--sort=desc`` for reverse order.
```

Comment:
```suggestion
    If you want to see translation output strings sorted alphabetically, add ``--sort=asc`` or ``--sort=desc`` option.```

Reply:
i wanted to do that at first but i think that my version reads better. am i wrong?

Reply:
Lets ask @javiereguiluz for another opinion 😃